Library/Homebrew/extend/git_repository.rb

```diff
@@ -43,16 +49,15 @@ def initialize(path)
       @path = path
     end
 
-    def send_command(*args)
+    def run(*args)
+      raise TapUnavailableError, path.name unless path.installed?
```

Comment:
Maybe this should be raised on initialization? Or perhaps we can include this in the `git_client` method, and then use a NullClient when it's not available?

Reply:
(I'll fix the test failure related to this once I have a clearer idea on where this should happen)